src/components/Header.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { Link } from 'react-router-dom';
+
+function Header() {
+  return (
+    <><Link to={'/'}>
```

Comment:
```suggestion
    <>
        <Link to={'/'}>
```

separamelo! que los fragment son unos solitarios! :D:D